fonts/css/rswebfonts-ie7-codes.css

```diff
@@ -222,7 +226,7 @@
 .icon-srv-content-management { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ab;&nbsp;'); }
 .icon-srv-data-storage { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ad;&nbsp;'); }
 .icon-srv-database { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ae;&nbsp;'); }
-.icon-srv-design-build { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9af;&nbsp;'); }
+.icon-srv-design-build-2 { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9af;&nbsp;'); }
```

Comment:
@brantdarilek We have to have both icon-srv-design-build and icon-srv-design-build-2 here to support some legacy icons.